routing/routing.go

```diff
@@ -183,6 +183,16 @@ func NewLBContext(r *http.Request, rt *Route) *LBContext {
 	}
 }
 
+func (c *LBContext) SkipEndpoint(index int) bool {
+	host := c.Route.LBEndpoints[index].Host
+	for i := range c.LBEndpoints {
+		if c.LBEndpoints[i].Host == host {
+			return false
+		}
+	}
+	return true
+}
```

Comment:
This is something consistent hash specific. Why does it leak into LBContext type?